packages/atclient/CMakeLists.txt

```diff
@@ -68,6 +69,7 @@ project(
  LANGUAGES C
 )
 
+set (CMAKE_C_STANDARD 99)
```

Comment:
This ensures that certain rules that are normally flagged by the LSP are not flagged, namely `Array designators are a C99 extension` which is a feature used in notification.h